HousingRegisterApi/V1/Services/CSVGeneratorService.cs

```diff
@@ -0,0 +1,115 @@
+using HousingRegisterApi.V1.Domain.FileExport;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Reflection;
+using System.Threading.Tasks;
+using System.Linq;
+
+namespace HousingRegisterApi.V1.Services
+{
+    public class CSVGeneratorService : ICSVService
```

Comment:
From a naming point of view, I would go for `CsvGeneratorService` to separate Csv from Generator